products/tarantool.md

```diff
@@ -0,0 +1,70 @@
+---
+permalink: /tarantool
+layout: post
+title: Tarantool
+command: $ tarantool --version
+releasePolicyLink: https://www.tarantool.io/en/doc/latest/release/policy/
+changelogTemplate: |
+  https://www.tarantool.io/en/doc/latest/release/__LATEST__/
+category: db
+iconSlug: NA
+eolColumn: Support Status
+releaseDateColumn: false
+sortReleasesBy: 'releaseCycle'
+releases:
+  - releaseCycle: "2.8"
+    release: 2021-12-22
+    eol: false
+    latest: "2.8.3"
+
+  - releaseCycle: "2.7"
+    release: 2021-08-19
+    eol: 2021-08-19
+    latest: "2.7.3"
+
+  - releaseCycle: "2.6"
+    release: 2021-04-21
+    eol: 2021-04-21
+    latest: "2.6.3"
+
+  - releaseCycle: "2.5"
+    release: 2020-12-30
+    eol: 2020-12-30
+    latest: "2.5.3"
+
+  - releaseCycle: "2.4"
+    release: 2020-10-22
+    eol: 2020-10-22
+    latest: "2.4.3"
+
+  - releaseCycle: "2.3"
+    release: 2020-07-17
+    eol: 2020-07-17
+    latest: "2.3.3"
+
+  - releaseCycle: "2.2"
+    release: 2019-12-31
+    eol: 2020-04-20
+    latest: "2.2.2"
```

Comment:
2.2.3 is the latest release in the 2.2 series.

Reply:
Fixed.

```diff
--- a/products/tarantool.md
+++ b/products/tarantool.md
@@ -16,6 +16,7 @@ releases:
     release: 2021-12-22
     eol: false
     latest: "2.8.3"
+    link: https://github.com/tarantool/tarantool/releases/tag/2.8.3
 
   - releaseCycle: "2.7"
     release: 2021-08-19
@@ -52,6 +53,7 @@ releases:
     eol: false
     lts: true
     latest: "1.10.12"
+    link: https://github.com/tarantool/tarantool/releases/tag/1.10.12
 
 ---
 
```

Reply:
Still 2.2.2 here.

Reply:
```diff
--- a/products/tarantool.md
+++ b/products/tarantool.md
@@ -47,7 +47,7 @@ releases:
   - releaseCycle: "2.2"
     release: 2019-12-31
     eol: 2020-04-20
-    latest: "2.2.2"
+    latest: "2.2.3"
 
   - releaseCycle: "1.10"
     release: 2021-12-22
```